.github/workflows/build.yml

```diff
@@ -81,6 +81,7 @@ jobs:
           path: /home/runner/go/pkg/mod
           key: GOMODCACHE-v2-${{ hashFiles('**/go.mod') }}
       - run: make controller
+      - run: make testapi
```

Comment:
minor - not needed - this can be pulled now

Reply:
It's needed when `testapi` has code changes.